nixos/modules/services/continuous-integration/github-runner.nix

```
@@ -300,20 +299,35 @@ in
         RestrictSUIDSGID = true;
         UMask = "0066";
         ProtectProc = "invisible";
-        ProcSubset = "pid";
         SystemCallFilter = [
-          "~@debug"
-          "~@mount"
-          "~@privileged"
+          "~@capset"
+          "~@clock"
           "~@cpu-emulation"
+          "~@module"
+          "~@mount"
           "~@obsolete"
+          "~@raw-io"
+          "~@reboot"
+          "~@setdomainname"
+          "~@sethostname"
```

Are those really system call sets? I cannot find those sets in systemd.exec(5). Maybe you meant to deny setdomainname(2) and sethostname(2)?

```suggestion
          "~setdomainname"
          "~sethostname"
```

---

Good catch, fixed, thanks!